client/asset/dcr/dcr.go

```diff
@@ -5299,9 +5299,13 @@ func (dcr *ExchangeWallet) StakeStatus() (*asset.TicketStakingStatus, error) {
 	if !dcr.connected.Load() {
 		return nil, errors.New("not connected, login first")
 	}
-	// Try to get tickets first, because this will error for RPC + SPV wallets.
+	// Try to get tickets first, because this will error for older RPC + SPV
+	// wallets.
 	tickets, err := dcr.tickets(dcr.ctx)
 	if err != nil {
+		if errors.Is(err, oldSPVWalletErr) {
+			return nil, nil
+		}
```

Comment:
In this case the UI doesn't show any information that the current version of dcrwallet is too low. Might be helpful to show that.

Reply:
Is there a reasonably easy way to do that? I don't expect the number of rpc+spv users will be high. If it's more that a few lines of work to properly communicate this message to the front end, I'm okay with how you have it, @JoeGruffins.